packages/assistify-help-request/server/hooks/searchHooks.js

```diff
@@ -0,0 +1,21 @@
+/* global SystemLogger */
+function onSpotlighStartHandler(item) {
+	SystemLogger.debug('onSpotlightStart', item.text);
```

Comment:
@lobo87 imho, there should be some analysis which determines the potentially accessible rooms (which you did in `onSpotlighEndHandler`) and propagate those to the search

Reply:
do you want me to fetch the rooms in here and return them? imho i would find this very confusing. this call back is implemented only to show how it can be used, for smarti the implementation has no real usecase.